engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html

```diff
@@ -5,7 +5,7 @@ <h3 translate ng-hide="selectedErrataOption === 'current'">Applicable Errata</h3
   <h3 translate ng-show="selectedErrataOption === 'current'">Installable Errata</h3>
 </div>
 
-<div data-extend-template="common/views/katello-agent-notice.html"></div>
+<div ng-show="host.content_facet_attributes.katello_agent_installed" data-extend-template="common/views/katello-agent-notice.html"></div>
```

Comment:
Here are the results I got, can you double check if this is what's expected?  Especially the bold ones (They may be correct but wanted to lay it out here just to be sure.)

### katello-agent status vs 4 banners
| - | `with_katello_agent` method return value | katello-agent installed? | banner status
| --- | --- | --- | --- |
| content-host-details-errata | true | yes | Present
content-host-details-package-actions | true | yes | Present
bulk-actions-errata modal | true | yes | Present
bulk-actions-package modal | true | yes | Present
| content-host-details-errata | true | no | __Present__
content-host-details-package-actions | true | no | __Present__
bulk-actions-errata modal | true | no | Not present
bulk-actions-package modal | true | no | Not present
| content-host-details-errata | false | yes | __Present__
content-host-details-package-actions | false | yes | __Present__
bulk-actions-errata modal | false | yes | Not present
bulk-actions-package modal | false | yes | Not present
| content-host-details-errata | false | no | Not present
content-host-details-package-actions | false | no | Not present
bulk-actions-errata modal | false | no | Not present
bulk-actions-package modal | false | no | Not present